dist/style.css

```diff
@@ -427,6 +455,88 @@
   color: #fff;
 }
 
+#contact {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+}
+
+.contact-text {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 327px;
+  height: 216px;
+  margin: 6rem 1rem 3rem 2rem;
+}
+.contact-text p {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.form {
+  display: none;
+}
+
+.form2 fieldset {
+  width: 328px;
+  height: 400px;
+  border: none;
+}
+.form2 fieldset input {
+  margin-bottom: 1rem;
+}
+.form2 fieldset input,
+.form2 fieldset textarea {
+  padding: 10px;
+  margin-left: 1rem;
+  width: 100%;
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+}
+.form2 fieldset input:focus,
+.form2 fieldset textarea:focus {
+  outline: none;
+}
+.form2 fieldset textarea {
+  background-color: #e7e5e5;
+}
+.form2 fieldset button {
+  width: 122px;
+  height: 48px;
+  background-color: #ff6b00;
+  color: #fff;
+  border: none;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  margin-left: 8em;
+  margin-top: 1rem;
+}
+
+.copyright {
+  border-top: 1px solid #DFE1E6;
+  margin-top: 8rem;
+}
+.copyright ul {
+  display: flex;
+  justify-content: center;
+}
+.copyright ul li {
+  margin: 15px;
+}
+.copyright ul li a {
+  font-size: 1.5em;
+  color: #505f79;
+}
+
 @media (min-width: 450px) and (max-width: 767px) {
   .intro {
     background-image: url("/images/hd.png");
```

Comment:
[OPTIONAL] please fix the background image url, as they are not displayed in the page, I think the path should be `url("../images/hd.png");` please do this for all the background images that are not displayed